### PR TITLE
Use new API

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,8 +33,135 @@
...
...nd not fully implemented. All help is welcome.\n\n## Why?\nBecause I wanted Ruby code completion in Atom and [atom-rsense](https://github.com/rsense/atom-rsense) hasn't been updated since June 2014, which is a long time for an Atom package. It's currently broken and I didn't like the dependency on Opal, as much as I enjoy Ruby, so I decided to create this.\n\n## Installation\nInstall required gems and Atom packages:\n```shell\n$ gem install rsense\n$ apm install autocomplete-plus\n```\n\n## Usage\nLaunch the rsense server in the project root you want completion:\n```shell\n$ rsense start\nrsense version: 0.5.18\nlogs at: /tmp/rsense.log\nprocess running at: 925\n```\n\n## Bugs\nA lot. If you're brave enough to try this out and notice any specifically, feel free to open an issue or even better submit a pull request.\n\n## Development\nClone the repository into your working directory:\n```shell\n$ git clone git@github.com:ianhattendorf/autocomplete-ruby.git\n```\n\nInstall dependencies:\n```shell\n$ cd autocomplete-ruby\n$ apm install\n```\n\nLink to Atom as a dev package:\n```shell\n$ apm link --dev\n```\n\nFeel free to fork it and submit a pull request for any changes you make.\n",
```
Comment (ianhattendorf) [inline package.json:40]: Do we need all of these extra fields? AFAIK npm/apm automatically locate README.md.

**Comment by martoko** [comment]
Is there any specific reason this hasn't been merged?

**Comment by ianhattendorf** [comment]
I never got a response from @vjudge1 regarding my comment above. His commit adds over 100 lines of cached info to package.json which shouldn't be there.

**Comment by martoko** [comment]
Seems reasonable. I might look into updating it to the new API, by looking at newer autocomplete packages, and not needlessly bloating the package.json.

Though I must say rsense and ruby autocompletetion for atom, seems pretty dead.

**Comment by ianhattendorf** [comment]
I agree, unfortunately without any updates to rsense there isn't much that can be done in this project. I've been moving away from ruby myself so I haven't spent much time looking for alternatives.

**Comment by martoko** [comment]
As of right now, I do not believe that there are any viable alternatives, which essentially means that there is pretty much no working autocompletion for ruby, except rubymine.

**Comment by Mexxerio** [comment]
@ianhattendorf I created a new PR for using the new API.

Haven't seen any viable alternatives to Rsense either. Kind of crazy to think that such a popular language doesn't have a good library for this. Ruby needs something like https://github.com/davidhalter/jedi

**Comment by ianhattendorf** [comment]
Thanks, I went ahead and merged your pull request. And I agree, Ruby hasn't been seeing much love on the IDE front.

**Pull request closed without merging**